x-pack/test_serverless/functional/test_suites/search/common_configs/config.group1.ts

```diff
@@ -13,13 +13,13 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
   return {
     ...baseTestConfig.getAll(),
     testFiles: [
-      require.resolve('../../common/home_page'),
+      // require.resolve('../../common/home_page'),
```

Comment:
these changes should not have been checked-in

Reply:
Oops. Missed to clean that up

Reply:
removed in https://github.com/elastic/kibana/pull/194857/commits/f9dbbe5d17e37c758da5403c4b94e497e3b9ec3a